### ISSUE TITLE
qques précisions dans le README

### ISSUE THREAD

**Issue by niconil** [cquest/osmfr-cartocss#39]
Je viens de dérouler la procédure

j'ai buté sur 2-3 petites choses dites-moi si cela mérite d'etre consigné dans le README (auquel cas je pourrais faire une proposition d'amendement)

en Debian (9.1) c'est la version postgresql 9.6 qui vient avec les packages
en Ubuntu (16.04LTS) c'est la version 9.5 
dans ce cas le script functions.sql doit etre adapté (j'ai vu cela dans #29 de mael)

dans le Readme on ne parle que de additions.sql
on pourrait lister préciséement les deux fichiers
- additions.sql
- functions.sql

dans additions.sql j'ai un doublon à la ligne 24/25 qui genere une erreur (sans gravité cependant)

ensuite que faire ? j'aurai aimé que le README m'accompagne un peu plus loin

un prérequis que je n'ai vu que dans un "issue" prcédent le #23 
je cite: "We need a carto version >= 0.16.3 to compile this style project.
The debian package install a 0.9.5."
idem pour le paquet Ubuntu

on s'en sort avec un
npm install -g @mapbox/carto
(et npm n'est pas dispo en standard sur debian stretch, mais on trouve)

en cherchant :
  $python -c 'import sys, yaml, json; json.dump(yaml.safe_load(sys.stdin), sys.stdout, indent=4, separators=(",", ": "))' < osmfr.yml > project.json.mml
  $carto -a "3.0.0" project.json.mml > mapnik.xml

à la compilation il rale sur tout les champ "name:" car prochainement marqué d'obsolescence
un petit update à faire sur osmfr.mml ?

voila sinon pourquoi pas faire une petite release la dernière visible est la 2.8.0
et date de 2014 ...

un merci a mael pour ses "issues" qui m'ont permis d'arriver jusqu'au bout (et a christian pour ses réponses)

**Comment by cquest** [comment]
Pour postgres, il me semble préférable de faire l'effort d'utiliser la dernière version pour bénéficier des améliorations (nombreuses et qui vont au delà de PARALLEL SAFE).
Le readme précise bien que c'est la 9.6 qui est requise.

**Comment by MaelREBOUX** [comment]
C'est en tout cas bien marqué dans https://github.com/cquest/osmfr-cartocss#dependencies
Donc c'est assumé que pour utiliser ce style tu **dois** avoir du PostGRESQL >= 9.6

Vu que sur une Ubuntu 16.04-3 LTS c'est toujours du PostGRESQL 9.5 il faut savoir modifier les sources pour récupérer cette dernière version. Pas à la portée du premier venu mais pas moins que mettre en place un serveur OSM ;)

**Comment by niconil** [comment]
et pour la release vous croyez que c'est possible afin d'avoir une certaine stabilité ?

pour le tag name le warning suggère de le remplacer par id déjà présent mais pas au meme niveau hiérarchique ... la je laisse les experts déterminer l'urgence de cette modification

en fait j'ai fait tout ça suite au dernier sotm-fr où j'ai annoncé publiquement mon intention de relancer une instance maposmatic. La compilation du theme osm-fr fait partie de la liste des choses à faire ...
c'est pour ça que je suis passé par là

merci à vous

**Comment by cquest** [comment]
Effectivement, des releases ça serait un plus, mais je n'ai jamais vraiment fonctionné comme jusqu'à maintenant. J'essaye de mettre ça au propre pour le prochaine mise à jour (fin d'année 2018 ?)
